src/net_processing.cpp

```diff
@@ -110,8 +110,11 @@ static constexpr auto GETDATA_TX_INTERVAL{60s};
 static const unsigned int MAX_GETDATA_SZ = 1000;
 /** Number of blocks that can be requested at any given time from a single peer. */
 static const int MAX_BLOCKS_IN_TRANSIT_PER_PEER = 16;
-/** Time during which a peer must stall block download progress before being disconnected. */
-static constexpr auto BLOCK_STALLING_TIMEOUT{2s};
+/** Default time during which a peer must stall block download progress before being disconnected.
+  * the actual timeout is increased temporarily if peers are disconnected for hitting the timeout */
+static constexpr auto DEFAULT_BLOCK_STALLING_TIMEOUT{2s};
+/** Maximum timeout for stalling block download. */
+static constexpr auto MAX_BLOCK_STALLING_TIMEOUT{64s};
```

Comment:
nit: Perhaps `BLOCK_STALLING_TIMEOUT_`{`DEFAULT`,`MAX`} would be better?

Reply:
Done, maybe that makes it easier to see that they are used for the same thing.